tests/commands/import.py

```diff
@@ -45,44 +45,43 @@ def test_import_emptyfile(capfd, afrikaans_tutorial, tmpdir):
 
 @pytest.mark.cmd
 @pytest.mark.django_db
-def test_import_onefile(capfd, afrikaans_tutorial, tmpdir):
+def test_import_onefile(capfd, tmpdir):
     """Load an one unit PO file"""
-    p = tmpdir.mkdir("sub").join("tutorial.po")
-    p.write("""msgid ""
-msgstr ""
-"X-Pootle-Path: /af/tutorial/tutorial.po\\n"
-"X-Pootle-Revision: %s\\n"
-
-msgid "rest"
-msgstr "test"
-           """ % (Revision.get() + 1))
+    from pootle_store.models import Store
+
+    p = tmpdir.mkdir("sub").join("store0.po")
+    store = Store.objects.get(pootle_path="/language0/project0/store0.po")
+    p.write(str(get_translated_storefile(store)))
     call_command('import', os.path.join(p.dirname, p.basename))
     out, err = capfd.readouterr()
-    assert "[update]" in err
-    assert "obsoleted 3" in err
-    assert "added 1" in err
-    assert "/af/tutorial/tutorial.po" in err
+    assert "/language0/project0/store0.po" in err
 
 
 @pytest.mark.cmd
 @pytest.mark.django_db
-def test_import_onefile_with_user(capfd, afrikaans_tutorial, tmpdir, member):
+def test_import_onefile_with_user(capfd, tmpdir, site_users):
     """Load an one unit PO file"""
-    p = tmpdir.mkdir("sub").join("tutorial.po")
-    p.write("""msgid ""
-msgstr ""
-"X-Pootle-Path: /af/tutorial/tutorial.po\\n"
-"X-Pootle-Revision: %s\\n"
-
-msgid "rest"
-msgstr "test"
-           """ % (Revision.get() + 1))
-    call_command('import', '--user=member', os.path.join(p.dirname, p.basename))
+    from pootle_store.models import Store
+
+    user = site_users['user'].username
+    p = tmpdir.mkdir("sub").join("store0.po")
+    store = Store.objects.get(pootle_path="/language0/project0/store0.po")
+    p.write(str(get_translated_storefile(store)))
+    call_command('import', '--user=%s' % user,
+                 os.path.join(p.dirname, p.basename))
     out, err = capfd.readouterr()
-    assert "[update]" in err
-    assert "obsoleted 3" in err
```

Comment:
@phlax, `allow_add_and_obsolete` flag has changed this behaviour and I've added tests for this functionality in import_export tests. I didn't want to test everything here.

> I tried to avoid testing the same stuff in different tests:
> 
> command tests check that command works;
> import tests check that allow_add_and_obsolete and file header processing stuff;
> store.update() tests check how different units are updated;